### ISSUE TITLE
Question: How to contribute? + AutoHotKey script for for shotcut commands

### ISSUE THREAD

**Issue by np-8** [markummitchell/engauge6#76]
Hello, 

I am new to Engauge, as well as github. Today I found this wonderful digitizer, but since it did not have options to define own shortcuts, I made them using [AutoHotKey](https://autohotkey.com/). Just to put my two cents, below is the code if anyone is interested. In addition, if some day I would like to help, where should I start?

### Autohotkey Script:
- Zoom in/out with Ctrl+Mouse Wheel
- WASD keys for moving in the window
- Q/E for zoom in/out
- Use 1,2,3,4,5 & 6 to select tools instead of Shift-F2, ... etc.

`#MaxHotkeysPerInterval 200
#HotkeyInterval 500

#IfWinActive, Engauge Digitizer

; Zoom in and out with control+mouse wheel
^WheelDown::Send {-}
^WheelUp::Send {+}

; WASD-keys for moving in the window
; and Q/E for zooming
a::SendInput !{WheelUp}
d::SendInput !{WheelDown}
w::SendInput {WheelUp}
s::SendInput {WheelDown}
q::Send {-}
e::Send {+}

; Shorcuts for the tools
1::SendInput +{F2}
2::SendInput +{F3}
3::SendInput +{F4}
4::SendInput +{F5}
5::SendInput +{F6}
6::SendInput +{F7}`

**Comment by markummitchell** [comment]
This definitely seems like a good thing for Windows users. I am wondering about the best way to make sure people can know it is available - maybe a link on the README.md file. Suggestions?

**Comment by crayxt** [comment]
We need to put this script to contrib directory and mention in README. 
@np-8 you will need to specify the license of your work - Public Domain is preffered for such things.

**Comment by np-8** [comment]
Let the licence be Public Domain then, or whatever is the licence of Engauge. I figured out that in engauge 7.0 there is some sort of mouse zooming, but I did not try that out since I am using this script. Feel free to use/modify/distribute any way you want. I also noticed that the script could be improved a little, since the A/D keys (left/right) will simulate Alt+MouseWheel to scroll horizontally, but for some reason the Alt there triggers the Windows hotkey for "Alt-F:File", "Alt-E: Edit".. etc. Also using Alt-E is not possible anymore since 'E' is used as a zoom hotkey. This could be improved by triggering the script for example only when Caps Lock is on.
